tools/generate_hotkeys.py

```diff
@@ -13,34 +17,135 @@
+                various_key_combinations = various_key_combination(
+                    key_combinations_list
+                )
+                mdFile.write(f"|{help_text}|{various_key_combinations}|\n")
+            mdFile.write("\n")
+
+    if not generate_temp_flag:
+        compare_temp_hotkeys_file(True)
+        with open(TEMP_OUTPUT_FILE, "r") as mdFile, open(
+            ORIGINAL_OUTPUT_FILE, "w"
+        ) as original_mdFile:
+            for line in mdFile:
+                original_mdFile.write(line)
+        TEMP_OUTPUT_FILE.unlink()
+        print(f"Hot Keys list saved in {ORIGINAL_OUTPUT_FILE}")
+
+
+def lint_hotkeys_file() -> None:
+    """
+    First compare temp_hotkeys and hotkeys file, if in sync then lint for key description and key duplication
+    """
+    generate_hotkeys_file(True)
+    compare_temp_hotkeys_file()
+    TEMP_OUTPUT_FILE.unlink()
+    # To lint keys description and duplicate keys
+    error_flag = 0
+    categories = read_help_categories()
+    for action in categories.keys():
+        check_duplicate_keys_list: List[str] = []
         for help_text, key_combinations_list in categories[action].items():
-            various_key_combinations = " / ".join(
-                [
-                    " + ".join([f"<kbd>{key}</kbd>" for key in key_combination.split()])
-                    for key_combination in key_combinations_list
-                ]
+            check_duplicate_keys_list.extend(key_combinations_list)
+            check_duplicate_keys_list = [
+                key for key in check_duplicate_keys_list if key not in KEYS_TO_EXCLUDE
+            ]
+            various_key_combinations = various_key_combination(key_combinations_list)
+            # Check description style
+            if not re.match(r"^[a-zA-Z\s/()',&@#:_-]*$", help_text):
+                print(
+                    f"Description - ({help_text}) for key combination - [{various_key_combinations}] should contain only alphabets, spaces and special characters except ."
+                )
+                error_flag = 1
+        # Check key combination duplication
+        duplicate_keys = [
+            key
+            for key in check_duplicate_keys_list
+            if check_duplicate_keys_list.count(key) > 1
+        ]
+        if len(duplicate_keys) != 0:
+            print(
+                f"Duplicate key combination for keys {duplicate_keys} for category ({HELP_CATEGORIES[action]})"
             )
-            mdFile.write(f"|{help_text}|{various_key_combinations}|\n")
-        mdFile.write("\n")
+            error_flag = 1
+    exit(error_flag)
 
-print(f"Hot Keys list saved in {ORIGINAL_OUTPUT_FILE}")
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="Lint and generate hotkeys")
+    parser.add_argument(
+        "--check-only",
+        action="store_true",
+        help="Check hotkeys file with key description and key duplication from keys file",
```

Comment:
This help text is a bit hard to read and understand. Maybe something like `"Lint hotkeys.md for key description style and key duplication"`?